Services/UserService/Interfaces/IDeleteUser.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Net;
+
+namespace IDeleteUserUserService.Interfaces
+{
+
```

Comment:
I don't know, mb it's a bug, but these empty lines look strange

Reply:
> I don't know, mb it's a bug, but these empty lines look strange

it is just empty; I wrote the code so